CMakeLists.txt

```diff
@@ -79,7 +84,7 @@ endif()
 # }
 
 # WITH_PLUGINS {
-if(WITH_LIBDL)
+if(WITH_LIBDL OR WIN32)
```

Comment:
Do we really want to use libdl and not LoadLibrary on cygwin win32?

Reply:
This line just turns the `WITH_PLUGINS` CMake flag on. All this does is add `rdkafka_plugin.c` to the build (see https://github.com/neptoess/librdkafka/blob/add-mingw-to-packaging/src/CMakeLists.txt#L82-L84). `rdkafka_plugin.c` relies on `rddl.c` for its functionality, and it looks like `rddl.c` properly detects `_WIN32` and handles plugin loading via `LoadLibrary`. If I'm misunderstanding this, let me know.